apps/easypid/src/locales/pt/messages.json

```
@@ -0,0 +1,3975 @@
+		"comments": ["Button label: toggles visibility of metadata attributes"],
+		"origin": [
+			["src/features/wallet/FunkeCredentialDetailAttributesScreen.tsx", 177]
+		],
+		"translation": "Ocultar atributos de metadados"
+	},
+	"optionSheet.hideMetadata": {
+		"message": "Hide metadata attributes",
+		"placeholders": {},
+		"comments": ["Option to hide metadata in option sheet"],
+		"origin": [
+			["src/features/share/FunkeRequestedAttributesDetailScreen.tsx", 179]
+		],
+		"translation": "Ocultar atributos de metadados"
+	},
+	"credentials.hideShareableAttributes": {
+		"message": "Hide shareable attributes",
+		"placeholders": {},
+		"comments": [
+			"Button label that toggles visibility of shareable attributes"
+		],
+		"origin": [
+			["src/features/wallet/FunkeCredentialDetailAttributesScreen.tsx", 160]
+		],
+		"translation": "Ocultar atributos partilháveis"
+	},
+	"common.fields.holder": {
+		"message": "Holder",
+		"placeholders": {},
+		"comments": [
+			"The holder identifier of a credential, usually a complex url structure"
+		],
+		"origin": [["../../packages/translations/src/commonMessages.ts", 436]],
+		"translation": "Titular"
+	},
+	"home.howDoesItWork": {
+		"message": "How does it work?",
+		"placeholders": {},
+		"comments": [],
+		"origin": [["src/features/wallet/FunkeWalletScreen.tsx", 100]],
+		"translation": "Como funciona?"
+	},
+	"onboardingWalletExplanation.slide3.title": {
+		"message": "How does it work?",
+		"placeholders": {},
+		"comments": ["Title for third slide of wallet onboarding"],
+		"origin": [["src/features/onboarding/screens/wallet-explanation.tsx", 58]],
+		"translation": "Como funciona?"
+	},
+	"didcommCancelFlow.issuance.description": {
+		"message": "If you decline, you will not receive the card.",
+		"placeholders": {},
+		"comments": ["Confirmation description when declining a credential offer"],
+		"origin": [["src/features/didcomm/utils.ts", 11]],
+		"translation": "Se recusar, não receberá o cartão."
+	},
+	"didcommCancelFlow.verification.description": {
+		"message": "If you stop, no data will be shared.",
+		"placeholders": {},
+		"comments": ["Confirmation description when cancelling a proof request"],
+		"origin": [["src/features/didcomm/utils.ts", 21]],
+		"translation": "Se parar, nenhum dado será partilhado."
+	},
+	"slideWizard.confirmation.stopSharingDescription": {
+		"message": "If you stop, no data will be shared.",
+		"placeholders": {},
+		"comments": [
+			"Description in confirmation dialog about stopping data sharing"
+		],
+		"origin": [["../../packages/app/src/components/SlideWizard.tsx", 55]],
+		"translation": "Se parar, nenhum dado será partilhado."
+	},
+	"didcommCancelFlow.connect.description": {
+		"message": "If you stop, nothing will be saved.",
+		"placeholders": {},
+		"comments": [
+			"Confirmation description when cancelling a connection request"
+		],
+		"origin": [["src/features/didcomm/utils.ts", 31]],
+		"translation": "Se parar, nada será salvo."
```

```suggestion
		"translation": "Se parar, nada será guardado."
```

Use "guardar" to translate "save a file". "Salvar" is Brazillian Portuguese.